src/BackupMetadata.cs

```diff
@@ -82,23 +82,39 @@ public class BackupMetadata : ICloneable
     /// </summary>
     public BackupFileTools.BackupType BackupType { get; set; }
 
+    internal BackupMetadata() { }
+
+    protected BackupMetadata(BackupMetadata backup)
+    {
+       CheckpointLsn = backup.CheckpointLsn;
+       DatabaseBackupLsn = backup.DatabaseBackupLsn;
+       DatabaseName = backup.DatabaseName;
+       FirstLsn = backup.FirstLsn;
+       LastLsn = backup.LastLsn;
+       PhysicalDeviceName = backup.PhysicalDeviceName;
+       ServerName = backup.ServerName;
+       StartTime = backup.StartTime;
+       BackupType = backup.BackupType;
+    }
+
     // used during testing
     public object Clone()
     {
       return MemberwiseClone();
     }
   }
 
-  public class StripedBackupSet
+  public class StripedBackupSet : BackupMetadata
   {
     public IEnumerable<BackupMetadata> StripedBackups { get; private set; }

```

Comment:
Can you rename `StripedBackupSet` to `StripedBackup`.  There is no 'set' collection here.  It's 1 entity with a list of different filenames.